src/freenet/l10n/freenet.l10n.de.properties

```diff
@@ -1356,8 +1382,10 @@ PluginManager.pluginLoadingFailedShort=Das Plugin ${name} konnte nicht geladen w
 PluginManager.pluginLoadingFailed=Das Plugin (Erweiterung) ${name} konnte nicht geladen werden.
 PluginManager.pluginLoadingFailedStillTryingOverFreenet=Es wird immer noch versucht, das Plugin über das Freenet zu laden.
 PluginManager.pluginLoadingFailedWithMessage=Das Plugin (Erweiterung) ${name} konnte nicht geladen werden: ${message}
-PluginManager.officialPluginLoadFailedSuggestTryAgain=Sie können es erneut versuchen.
+PluginManager.officialPluginLoadFailedSuggestTryAgainFreenet=Sie können versuchen, dieses Plugin erneut über Freenet herunterladen, oder es über das Web probieren.
+PluginManager.officialPluginLoadFailedSuggestTryAgainHTTPS=Sie können dieses Plugin über Freenet herunterladen, oder es noch einmal über das Web versuchen.
 PluginManager.officialPluginLoadFailedTryAgainFreenet=Lade das Plugin aus dem Freenet
+PluginManager.officialPluginLoadFailedTryAgain=Neuer Versuch, das offizielle Plugin von den Freenet-Webservern herunterzuladen (NICHT ANONYM!)
```

Comment:
This key doesn’t exist anymore in the source translation. I remember removing it! 😀

Reply:
fixed - thank you!